elements/lisk-dpos/test/utils/round_delegates.ts

```diff
@@ -53,7 +53,7 @@ export const delegateAccounts = delegatePublicKeys.map(
 		const votedDelegatesPublicKeys = [
 			votedDelegates[index].publicKey,
 			votedDelegates[index + 1].publicKey,
-			votedDelegates[index % 10].publicKey,
```

Comment:
just to understand, in the first place why did we chose to do mod 10?

Reply:
it was originally like that. i think it just needs some random delegate public key, but it has to be distinct